MediaBrowser.Model/MediaBrowser.Model.csproj

```diff
@@ -13,6 +13,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="2.2.0" />
```

Comment:
`Microsoft.AspNetCore.Http` should be replaced with `Microsoft.AspNetCore.Http.Abstractions`.
I'd not include `Microsoft.AspNetCore.Http.Features` if at all possible.
Remember that EVERY other Jelyfin c# project includes this one.